site/global-guides/voice/authentication.mdx

```diff
@@ -0,0 +1,75 @@
+- **Headers**
+  - **Authorization** - This request requires the request to be authenticated. You MUST provide your `ClientID` and `ClientSecret` in a Basic Authorization header. Basic Authentication is a Base64 encoded string that contains the username and password.
+  - **Content-type** - For this request we **MUST** to use `application/x-www-form-urlencoded`. This will urlencoded our parameters making them into `key=value` pairs which will be separated by a `&`.
+- **Body** - There is only one piece of data that must be transmitted in the form and that is the `grant_type` currently we only support `client_credentials` OAuth2 `grant_type`.
+
+### Response
+A sample response is:
+```
+{
+	"access_token": "eyJraWQiOiJzZ25tLTk1ZTE0YTFiLTRiODItNDM5Zi1hYjhmLTYwMmI5MzYwMjZkZiIsImFsZyI6IlJTMjU2In0..kl3HYFQIcP4xZd78wUkdXxntxBu1d7b5ZCt99qVObC-gUZd5KSq2J7Q5rDb2LjP1_WVndcLQSqCoq3Anvp03hJWM6mamZL3dWHxjGLwkrIlzmNx9ZFGhTGIEsYLyaQqy8MonWYw2mJ4Z0APEfTVbTBHNIGrm_9GT6rIkdOwQFM-XgH1Tau4JbpFJun3n6o15WTgjzdEj9fIwd385CPwL-pAmOiozbYWUEzgqkXjSGHI11hiLDu-tv_8Ds06Cx4iCnL1F6_dFrnpD3CF0i6JJYVrvLmi6vgzoxp9YEIRBwaOZTSuYuYt03SQfbMZy8L2Z71sRKLLGt3TrxgtwyjefpQ",
+	"access_token_id": "a5xA3xMKEggGwvpSLtk2lRb",
+	"token_type": "Bearer",
+	"expires_in": 600
+}
+```
+
+Let's see what each item in the response object is meant for:
+- `access_token`: This is the JSON Web Token (JWT) that will be passed in the Authorization header of the requests to the In-App Calling API.
+- `access_token_id`: This will be the ID of tha access token to prevent tokens from being re-used. It will be the same as the jti claim within the access token.
+- `token_type`: This is used to identify the type of token and also the prefix to be used in the Authorization header.
+- `expires_in`: This is how long in seconds the token will be valid. In the example above the value is `600`, this means the token will be valid for 600 seconds or 10 mins. You **SHOULD** use the token until 10-30 seconds before expiration at which point you should request a new token in the same manner as described above.
+
+### Using the Access Token
+
+Once you have the token you can now perform authenticated requests to the WebRTC Gateway, all you need to do is add an `Authorization` header to the request with a value of `<token_type> <access_token>`.
+
+Using the example response above a request would look similar to:
+```sh
+curl --request GET \
+  --url https://<webrtc_gateway>/some/protected/path \
```

Comment:
Not sure what or how we described the `webrtc_gateway` value.

Reply:
I am not sure the of the URL perhaps we can get this from Pritom or Preo.

Reply:
I think it's `sbc.webrtc-app.bandwidth.com`.
@vikash-band What is the correct URL?

Reply:
sbc.webrtc-app.bandwidth.com is correct url